docs/memcached.md

```diff
@@ -0,0 +1,64 @@
+# `memcached` Metrics Receiver
+
+The memcached receiver can retrieve stats from your memcached server through the . 
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your memcached configuration.
+
+To configure a receiver for your memcached metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `memcached`. |
+| `endpoint`            | `localhost:3306`          | The url, or unix socket file path, for your memcached server. |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    redis_metrics:
+      type: memcached
+      address: localhost:3306
+      collection_interval: 30s
+  service:
+    pipelines:
+      memcached_pipeline:
+        receivers:
+          - memcached_metrics
+```
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| memcached.bytes | Current number of bytes used by this server to store items. | By | Gauge | <ul> </ul> |
+| memcached.commands | Commands executed. | 1 | Sum | <ul> <li>command</li> </ul> |
+| memcached.connections.current | The current number of open connections. | connections | Sum | <ul> </ul> |
```

Comment:
`current` should be a Gauge (or an UpDownCounter if we're using that terminology in these docs) since it can both increase and decrease.

Reply:
Updated to the new metric types after update.